include/alpaka/core/OmpSchedule.hpp

```diff
@@ -27,33 +27,33 @@ namespace alpaka
         //! whether OpenMP is enabled.
         struct Schedule
         {
-            //! Special schedule setting which does not change the internal control variables of OpenMP
-            constexpr static std::uint32_t NoSchedule = 0u;
-            //! Integers corresponding to the mandatory OpenMP omp_sched_t enum values as of version 5.0.
-            constexpr static std::uint32_t Static = 1u;
-            constexpr static std::uint32_t Dynamic = 2u;
-            constexpr static std::uint32_t Guided = 3u;
-            constexpr static std::uint32_t Auto = 4u;
-            //! Each schedule value can be combined with monotonic using + or |.
-            constexpr static std::uint32_t Monotonic = 0x80000000u;
+            //! Schedule kinds corresponding to arguments of OpenMP schedule clause
+            //!
+            //! Kinds also present in omp_sched_t enum have the same integer values.
+            //! It is enum, not enum class, for shorter usage as omp::Schedule::[kind]
+            enum Kind
```

Comment:
please use `enum class`

Reply:
I think it makes usage more wordy in this case, and also will break the 0.6.0 interface

Reply:
Ah, OK. Not breaking the interface is a good reason